### PR TITLE
Double mobile execution implementation

### PR DESCRIPTION
**Pull request by angy113**

### CODE REVIEW

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/components/BottomMenu.java [hunk 1]
```diff
@@ -19,13 +19,21 @@
     private WebElement btnCalls;
 
     /**
+     * Initialize an instance of {@link BottomMenu}.
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/components/BottomMenu.java]: Change the verb to third person: Initializes

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/BaseAppLayout.java [hunk 2]
```diff
@@ -15,24 +15,28 @@
...
     private BottomMenu bottomMenu;
 
-    @FindBy(xpath = "//android.widget.TextView[@text='Wifi']")
+    @FindBy(xpath = "//android.widget.LinearLayout[@resource-id='com.idtmessaging.koala:id/search_view']"
+            + "/preceding-sibling::android.widget.FrameLayout")
     private WebElement btnSettings;
 
     /**
-     * Clicks on settings icon in the top menu.
+     * Initialize an instance of {@link BaseAppLayout}.
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/BaseAppLayout.java]: Change the verb to third person: Initializes

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Call/CallInCourseLayout.java [hunk 3]
```diff
@@ -14,12 +14,22 @@
     private WebElement btnEndCall;
 
     /**
+     * This builds initialize the AbstractBasePage.
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Call/CallInCourseLayout.java]: Update the doc to 
Initializes an instance of ....

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Call/CallsLayout.java [hunk 4]
```diff
@@ -47,6 +47,16 @@
     private WebElement btnFreeCall;
 
     /**
+     * This builds initialize the AbstractBasePage.
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Call/CallsLayout.java]: Update the doc to
Initializes an instance of ....

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Call/FeedBackCallLayout.java [hunk 5]
```diff
@@ -14,14 +14,24 @@
     private WebElement btnClose;
 
     /**
+     * This builds initialize the AbstractBasePage.
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Call/FeedBackCallLayout.java]: Update the doc to
Initializes an instance of ....

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/CountryListLayout.java [hunk 6]
```diff
@@ -20,13 +20,23 @@
     private List<WebElement> tvCountry;
 
     /**
+     * This builds initialize the AbstractBasePage.
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/CountryListLayout.java]: Update the doc to
Initializes an instance of ....

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/HubLayout.java [hunk 7]
```diff
@@ -12,6 +12,16 @@
     private WebElement btnStartConversation;
 
     /**
+     * This builds initialize the AbstractBasePage.
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/HubLayout.java]: Update the doc to
Initializes an instance of ....

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Login/LoginLayout.java [hunk 8]
```diff
@@ -29,13 +30,23 @@
     private WebElement etPhoneNumber;
 
     /**
+     * This builds initialize the AbstractBasePage.
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Login/LoginLayout.java]: Update the doc to
Initializes an instance of ....

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Login/PinVerificationLayout.java [hunk 9]
```diff
@@ -33,6 +33,16 @@
     private WebElement btnResend;
 
     /**
+     * This builds initialize the AbstractBasePage.
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Login/PinVerificationLayout.java]: Update the doc to
Initializes an instance of ....

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Profile/ProfileLayout.java [hunk 10]
```diff
@@ -38,12 +38,22 @@
     private WebElement btnSave;
 
     /**
+     * This builds initialize the AbstractBasePage.
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Profile/ProfileLayout.java]: Update the doc to
Initializes an instance of ....

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Settings/SettingsLayout.java [hunk 11]
```diff
@@ -18,13 +18,23 @@
     private WebElement btnSupport;
 
     /**
+     * This builds initialize the AbstractBasePage.
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Settings/SettingsLayout.java]: Update the doc to
Initializes an instance of ....

File: IDT.CallingApp/src/test/java/com/jalasoft/callingapp/steps/SettingsAndInfoSteps.java [hunk 12]
```diff
@@ -27,10 +27,12 @@ public SettingsAndInfoSteps() {
 
     /**
      * It goes to settings section.
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/test/java/com/jalasoft/callingapp/steps/SettingsAndInfoSteps.java]: I think that it should be better use: Goes to....

File: src/main/java/com/jalasoft/ui/layouts/BaseLayout.java [hunk 13]
```diff
@@ -14,12 +14,18 @@
 
     protected WebDriverWait wait;
 
+    protected int number;
+
     /**
      * This builds initialize the AbstractBasePage.
```
Comment (wMercadoJala) [inline src/main/java/com/jalasoft/ui/layouts/BaseLayout.java]: Change to Initializes to ..

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/components/BottomMenu.java (hunk 1 above)
Comment (angy113) [reply IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/components/BottomMenu.java]: Done

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/BaseAppLayout.java (hunk 2 above)
Comment (angy113) [reply IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/BaseAppLayout.java]: Done

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Call/CallInCourseLayout.java (hunk 3 above)
Comment (angy113) [reply IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Call/CallInCourseLayout.java]: Done

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Call/CallsLayout.java (hunk 4 above)
Comment (angy113) [reply IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Call/CallsLayout.java]: Done

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Call/FeedBackCallLayout.java (hunk 5 above)
Comment (angy113) [reply IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Call/FeedBackCallLayout.java]: Done

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/CountryListLayout.java (hunk 6 above)
Comment (angy113) [reply IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/CountryListLayout.java]: Done

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/HubLayout.java (hunk 7 above)
Comment (angy113) [reply IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/HubLayout.java]: Done

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Login/LoginLayout.java (hunk 8 above)
Comment (angy113) [reply IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Login/LoginLayout.java]: Done

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Login/PinVerificationLayout.java (hunk 9 above)
Comment (angy113) [reply IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Login/PinVerificationLayout.java]: Done

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Profile/ProfileLayout.java (hunk 10 above)
Comment (angy113) [reply IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Profile/ProfileLayout.java]: Done

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Settings/SettingsLayout.java (hunk 11 above)
Comment (angy113) [reply IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Settings/SettingsLayout.java]: Done

**Pull request merged**